combiner/combiner.go

```diff
@@ -0,0 +1,208 @@
+package combiner
+
+import (
+	"runtime"
+	"sync"
+
+	"github.com/philippgille/gokv"
+)
+
+// Check we satisfy interface.
+var _ gokv.Store = &Store{}
+
+// Store is a `gokv.Store` implementation that forwards its
+// calls to other `gokv.Store`s.
+type Store struct {
+	backends []gokv.Store
+	parallel bool
+}
+
+// A helper type to check for errors when creating `gokv.Store`s.
+type Backend struct {
```

Comment:
Memo to myself: Maybe rename to `Target`. Store and backend might be ambiguous - a package user might not immediately understand the difference. Target fits the description that the combiner _forwards_ the actions to its... targets. On the other hand backend is common in some databases that implement for example the Redis protocol and offer multiple _backends_ as storage (e.g. RocksDB, bbolt, ...).